### PR TITLE
29 contacts feature delete and update others features

### PR DESCRIPTION
**Pull request by Cxristian**
create feature delete

### CODE REVIEW

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicPage.java [hunk 1]
```diff
@@ -173,13 +173,25 @@ public ContactFormAbstract clickNewContact() {
...
     }
 
+    /**
+     * Check name in contact List.
+     *
+     * @param name string.
+     * @return boolean.
+     */
+    @Override
+    public boolean checkContactList(String name) {
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicPage.java]: It method should be verified with assert in the step

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 2]
```diff
@@ -28,11 +28,8 @@
 import org.fjala.gugumber.salesforce.ui.PageLayoutConfig;
 import org.fjala.gugumber.salesforce.ui.PageLayoutFactory;
 import org.fjala.gugumber.salesforce.ui.PageLayoutType;
-import org.fjala.gugumber.salesforce.ui.pages.contact.ContactDetailsAbstract;
-import org.fjala.gugumber.salesforce.ui.pages.contact.ContactFormAbstract;
-import org.fjala.gugumber.salesforce.ui.pages.contact.ContactLightningProfilePage;
-import org.fjala.gugumber.salesforce.ui.pages.contact.ContactPageAbstract;
-import org.fjala.gugumber.salesforce.ui.pages.contact.ContactProfilePageAbstract;
+import org.fjala.gugumber.salesforce.ui.pages.contact.*;
```
Comment (AKatrin) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: You should usage * in imports when the imports of package o class is greater than 10

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java [hunk 3]
```diff
@@ -60,6 +66,15 @@ public String getFullNameTitleContact() {
         return contactTitle.getText();
     }
 
+    /**
+     * Delete contact.
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java]: Document should start in third person

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java [hunk 4]
```diff
@@ -60,6 +66,15 @@ public String getFullNameTitleContact() {
...
+     */
+    @Override
+    public void deleteContact() {
+        deleteBtn.click();
+        driver.switchTo().alert().accept();
+    }
+
     /**
      * Check the section Details.
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java]: Documentation should start in thir person

Comment (AKatrin) [reply src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java]: Fix documentation an name method

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningForm.java [hunk 5]
```diff
@@ -19,6 +19,8 @@
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 
+import java.util.Date;
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningForm.java]: order this import please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactProfilePageAbstract.java [hunk 6]
```diff
@@ -45,4 +45,9 @@
     public String getIdFromUrl() {
         return UrlComponents.splitUrlToId(driver.getCurrentUrl());
     }
+
+    /**
+     * Delete Contact in contact page.
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactProfilePageAbstract.java]: Comment this line in third person please

**Pull request merged**